app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt

```diff
@@ -104,7 +104,7 @@ class MessageActionsDialog(
                 hasUserActorId(message) &&
                 currentConversation?.type != ConversationType.ROOM_TYPE_ONE_TO_ONE_CALL
         )
-        initMenuDeleteMessage(showMessageDeletionButton)
+        initMenuDeleteMessage(ChatMessage.MessageType.SYSTEM_MESSAGE != message.getCalculateMessageType())
```

Comment:
this needs to be reverted to be `showMessageDeletionButton` again so the logic from ChatActivity to show/hide the deletion button is used.
https://github.com/nextcloud/talk-android/blob/288ec688349dd55e885c25f330c79c21e232277a/app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt#L4172

Reply:
Done